app/connector/activemq/src/main/resources/META-INF/syndesis/connector/activemq.json

```diff
@@ -93,7 +93,7 @@
       "defaultValue": "false",
       "deprecated": false,
       "secret": false,
-      "labelHint": "For convenience in only development environments. Do not set for secure production environments.",
+      "labelHint": "For convenience in only development environments. Ensure certificate checks are enabled for secure production environments.",
```

Comment:
This has the potential to be misleading because the field name and the hint imply that checking certificates is for convenience in only development environments. This is what the user will see: 
Check Certificates
For convenience in only development environments.

So I think the hint has to be: "Ensure certificate checks are enabled for secure production environments. Disable for convenience in only development environments."

Reply:
Good call, I'll make this change now!